avoidance_dis_max: 1.2
avoidance_dis_min: 0.6
